events_user.go

```diff
@@ -100,6 +100,21 @@ type EventNewKeyTypeData struct {
 	Fingerprint string `json:"fingerprint"`
 }
 
+type EventEditKeyTypeData struct {
+	Name        string `json:"name"`
+	Fingerprint string `json:"fingerprint"`
+}
+
+type EventEditKeyType struct {
+	Event
+	TypeData EventEditKeyTypeData `json:"type_data"`
+}
+
+func (ev *EventEditKeyType) String() string {
+	d := ev.TypeData
+	return fmt.Sprintf("Key '%s' edited on %s app", d.Name, ev.AppName)
```

Comment:
I may be confused by what is the edit key event, but I thought it is created when editing a SSH key in my profile, isn't it? Hence I don't get the reference to the app name here